models/bamf_ct_liver_tumor/utils/NNUnetRunnerV2.py

```diff
@@ -65,51 +54,26 @@ def task(self, instance: Instance, in_data: InstanceData, out_data: InstanceData
         out_dir = self.config.data.requestTempDir(label="nnunet-model-out")
         os.environ['nnUNet_results'] = out_dir
 
-        # symlink nnunet input folder to the input data with python
         # create symlink in python
-        # NOTE: this is a workaround for the nnunet bash script that expects the input data to be in a specific folder
-        #       structure. This is not the case for the mhub data structure. So we create a symlink to the input data
+        # NOTE: this is a workaround for the nnunet bash script that expects the model files to be in a output folder
+        #       This is not the case for the mhub data structure. So we create a symlink to the input data
         #       in the nnunet input folder structure.
-        # os.symlink(os.environ['WEIGHTS_FOLDER'], os.path.join(out_dir, 'nnUNet'))
         os.symlink(os.path.join(os.environ['WEIGHTS_FOLDER'], self.nnunet_dataset), os.path.join(out_dir, self.nnunet_dataset))
-        
-        # NOTE: instead of running from commandline this could also be done in a pythonic way:
-        #       `nnUNet/nnunet/inference/predict.py` - but it would require
-        #       to set manually all the arguments that the user is not intended
-        #       to fiddle with; so stick with the bash executable
 
         # construct nnunet inference command
         bash_command  = ["nnUNetv2_predict"]
         bash_command += ["-i", str(inp_dir)]
         bash_command += ["-o", str(out_dir)]
         bash_command += ["-d", self.nnunet_dataset]
         bash_command += ["-c", self.nnunet_config]
-        
-        # add optional arguments
-        if self.folds is not None:
-            bash_command += ["-f", str(self.folds)]
-
-        if not self.use_tta:
-            bash_command += ["--disable_tta"]
 
         self.v(f" > bash_command:     {bash_command}")
         # run command
         self.subprocess(bash_command, text=True)
 
-        # output meta
-        meta = {
-            "model": "nnunet",
-            "nnunet_dataset": self.nnunet_dataset,
-            "nnunet_config": self.nnunet_config,
-            "roi": self.roi
-        }
-
```

Comment:
Since we change the implementation from dynamic to static we can add the metadata statically in the `@IO.Output`decorator:

```python
@IO.Output("out_data", 'VOLUME_001.nii.gz', 'nifti:mod=seg:model=nnunet:nnunet_dataset= Dataset006_Liver:nnunet_config=3d_fullres:roi=SPLEEN,KIDNEY,GALLBLADDER,DUODENUM,PANCREAS,SMALL_INTESTINE,LUNG,LIVER,LIVER+NEOPLASM',
               data='in_data', the="output data from nnunet")
```

Reply:
Thank you! I have updated the logic